sss/ut/sss_ut.c

```diff
@@ -85,7 +83,8 @@ extern struct m0_fop_type m0_fop_process_fopt;
 static void rpc_client_and_server_start(void)
 {
 	int rc;
-
+	struct m0_net_xprt *xprt = m0_net_xprt_default_get();
+	sctx.rsx_xprts = &xprt;
```

Comment:
Storing the address of a local variable in a static structure?

Reply:
incorporated